app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self._conn = sqlite3.connect("cinema.sqlite")
+        self._table_name = "actors"
+
+    def create(
+            self,
+            first_name: str,
+            last_name: str
+    ) -> None:
+        self._conn.execute(
+            f"INSERT INTO {self._table_name} "
+            "(first_name, last_name) VALUES (?, ?) ",
+            (first_name, last_name)
+        )
+        self._conn.commit()
+
+    def all(self) -> None:
+        actors_cursor = self._conn.execute(
+            f"SELECT * FROM {self._table_name}"
+        )
+        for row in actors_cursor:
+            print(row)
```

Comment:
all - a method that returns a list of Actor instances from DB (read README)

Reply:
Good point, thanks)